launch/Jenkinsfile

```
@@ -68,7 +68,7 @@ def releaseOpenHabComponent(componentName, branch, releaseVersion, nextVersion,
                 def mvnSnapshotOptions = ""
                 def mvnOptionsMap = [:]
                 
-                mvnOptionsMap["build"] = '-DreleaseArgs=skipTests=true,skipChecks=true -DskipChecks=true -DskipTests=true -DgenerateBackupPoms=false' //
+                mvnOptionsMap["build"] = '-DreleaseArgs=skipTests=true,skipChecks=true -DskipChecks=true -DskipTests=true -DgenerateBackupPoms=false -Dspotless.check.skip=true' //
```

It probably also needs to be added to the releaseArgs:

```suggestion
                mvnOptionsMap["build"] = '-DreleaseArgs=skipTests=true,skipChecks=true,spotless.check.skip=true, -DskipChecks=true -DskipTests=true -DgenerateBackupPoms=false -Dspotless.check.skip=true' //
```